lemminx-maven/pom.xml

```diff
@@ -130,15 +130,20 @@
 			<version>${maven.version}</version>
 			<scope>compile</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.eclipse.aether</groupId>
-			<artifactId>aether-connector-basic</artifactId>
-			<version>1.1.0</version>
-		</dependency>
 		<dependency>
 			<groupId>io.takari.aether</groupId>
 			<artifactId>aether-connector-okhttp</artifactId>
 			<version>0.17.8</version>
+			<exclusions>

```

Comment:
Can you please add a comment mentioning the issue; so we easily know later why this shouldn't be modified blindly?

Reply:
Done